content/en/docs/migrating/_index.md

```diff
@@ -0,0 +1,819 @@
+[session.Must]({{< apiref v1="aws/session#Must" >}}) must be migrated to
+[config.LoadDefaultConfig]({{< apiref "config#LoadDefaultConfig" >}}).
+
+The `config` package provides several helper functions that aid in overriding the shared configuration loading
+programmatically. These function names are prefixed with `With` followed by option that they override. Let's look at
+some examples of how to migrate usage of the `session` package. For more information on loading shared configuration
+see [Configuring the {{% alias sdk-go %}}]({{% ref "configuring-sdk" %}}).
+
+#### Examples
+#####  Migrating from NewSession
+In this example usage of `session.NewSession` without additional argument parameters is migrated to
+`config.LoadDefaultConfig`.
+
+###### V1
+
+```go
+import "github.com/aws/aws-sdk-go/aws/session"
+
+// ...
+
+sess, err := session.NewSession()
+if err != nil {
+	// handle error
+}
+```
+
+###### V2
+```go
+import "github.com/aws/aws-sdk-go-v2/config"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig()
+if err != nil {
+	// handle error
+}
+```
+
+##### Migrating from NewSession with aws.Config options.
+In this example we will show how to migrate overriding of `aws.Config` values during configuration loading. One or more
+`config.With*` helper functions can be provided to `config.LoadDefaultConfig` to override the loaded configuration
+values. In this example the AWS Region is overridden to `us-west-2` using the
+[config.WithRegion]({{< apiref "config#WithRegion" >}}) helper function.
+
+###### V1
+```go
+import "github.com/aws/aws-sdk-go/aws"
+import "github.com/aws/aws-sdk-go/aws/session"
+
+// ...
+
+sess, err := session.NewSession(aws.Config{
+	Region: aws.String("us-west-2")
+})
+if err != nil {
+	// handle error
+}
+```
+
+###### V2
+```go
+import "github.com/aws/aws-sdk-go-v2/config"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig(config.WithRegion("us-west-2"))
+if err != nil {
+	// handle error
+}
+```
+
+#####  Migrating from NewSessionWithOptions
+In this example we will show how to migrate overriding values during configuration loading. One or more
+`config.With*` helper functions can be provided to `config.LoadDefaultConfig` to override the loaded configuration
+values. In this example we show how to override the target profile that is used when loading the AWS SDK shared
+configuration.
+
+###### V1
+```go
+import "github.com/aws/aws-sdk-go/aws"
+import "github.com/aws/aws-sdk-go/aws/session"
+
+// ...
+
+sess, err := session.NewSessionWithOptions(aws.Config{
+	Profile: "my-application-profile"
+})
+if err != nil {
+	// handle error
+}
+```
+
+###### V2
+```go
+import "github.com/aws/aws-sdk-go-v2/config"
+
+// ...
+
+cfg, err := config.LoadDefaultConfig(config.WithSharedConfigProfile("my-application-profile"))
+if err != nil {
+	// handle error
+}
+```
+
+## Credentials & Credential Providers
+
+The [credentials]({{< apiref v1="aws/credentials" >}}) package and associated credential providers have been relocated
```

Comment:
Should the text of `[credentials`] here be `aws/credentials`?. Not sure if that would help add clarity since a nested package was moved to the root of the SDK.